README.md

```diff
@@ -115,13 +115,64 @@ Note that `called` can be used as a chainable method.
 
 #### .with
 
-Assert that a spy has been called with a given argument.
+Assert that a spy has been called with a given argument at least once,
+even if more arguments were provided.
 
 ```js
 expect(spy).to.have.been.called.with('foo');
 spy.should.have.been.called.with('foo');
 ```
 
+Will pass for `spy('foo')`, `spy('foo', 'bar')`, `spy(); spy('foo')`,
+etc.
+
+Can be used for multiple arguments:
+
+```js
+spy('foo');
+spy('bar');
+expect(spy).to.have.been.called.with('bar', 'foo');
+// or spy.should.have.been.called.with('bar', 'foo');
+// Either will pass.
```

Comment:
Nitpick: Just for consistency with the other examples, I'd say 135 shouldn't be commented and 136 should be removed.